docs/central-submissions.rst

```diff
@@ -330,3 +330,57 @@ You can make an Enketo form offline-capable by changing the form's web address a
     #. Press ``Enter`` on your keyboard, and make sure that you see an orange connectivity icon in the upper left of the form.
 
     #. Copy the link and share it with your data collector(s)
+
+Customizing end-of-form behavior
+--------------------------------
+
+.. tab-set::
+   
+   .. tab-item:: Central v2025.2.3+
+   
+      For Public Access Links, the default end-of-form behavior displays a generic thank you message. To redirect respondents to a custom page, provide a `return_url`. Additionally, to enable respondents or data collectors to make multiple submissions without refreshing the page after each submission, append `&single=false` to the Public Access link:
```

Comment:
Public Access Links could be bold to match Data Collectors below or there could be headings for each. Not critical.